.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,7 @@ _put `n/a` if item is not relevant to PR changes_
 
 - [ ] Has the CHANGELOG been updated?
 - [ ] Has the README been updated?
+- [ ] Has the CONTRIBUTING been updated?
```

Comment:
Minor: This sounds odd. Maybe use `Has the CONTRIBUTING doc been updated` instead.

Reply:
True. Fixed